### ISSUE TITLE
 implicit function instantiation requires go1.18 or later

### ISSUE THREAD

**Issue by KaptinLin** [gookit/validate#220]
**System (please complete the following information):**

 - OS: `windows` 
 - GO Version: `1.21 rc2` 
 - Pkg Version: `1.1.1` 

**Describe the bug**

go\pkg\mod\github.com\gookit\validate@v1.4.6\messages.go:458:16: implicit function instantiation requires go1.18 or later (-lang was set to go1.15; check go.mod)

**Comment by KaptinLin** [comment]
it's fixed when i install directly from github commit 

```
go get -u github.com/gookit/validate@5dfa4ac530b19331700d71e38b5ff1da19aa88c9
```

**Comment by inhere** [comment]
hi all, has been released the `v1.5.0`.
